jekyll/_cci2/custom-images.md

```diff
@@ -16,10 +16,28 @@ This document describes how to create and use custom Docker images with CircleCI
 
 ## Overview
 
-CircleCI 2.0 gives you access to the power and flexibility of Docker. One of the ways you can take advantage of this is to create custom Docker images for your jobs. Following are the benefits of creating a custom image:
+Docker is a first-class citizen in CircleCI
+and provides you with a powerful way
+to specify dependencies for your projects.
+If CircleCI's [convenience images]({{ site.baseurl }}/2.0/circleci-images/) do not suit your needs,
+consider creating a custom Docker image for your jobs.
+There are two major benefits
+of doing this:
+
+- **Faster job execution.**
+Packaging your required tools into a custom image
+removes the need
+to install them for every job.
+
+- **Cleaner configuration.**
+Baking lengthy installation scripts into a custom image
```

Comment:
Baking->Adding/clutter->lines (these are colloquial, use plain technical terms for ease of understanding by non-native speakers and for localization).